collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -40,7 +48,7 @@ def __init__(self, configuration=None, name=None):
         self.sm = self.configuration.get('status_map', DEFAULT_STATUS_MAP)
         self.order, self.definitions = charts_template(self.sm)
         self.url = self.configuration.get('url', DEFAULT_URL)
-        self.collected_alarms = set()
+        self.show_alarm_values = bool(self.configuration.get('show_alarm_values', DEFAULT_SHOW_ALARM_VALUES))
```

Comment:
```suggestion
        self.сollect_alarm_values = bool(self.configuration.get('collect_alarm_values', DEFAULT_SHOW_ALARM_VALUES))
```

I think `collect_` is better because it literally reflects the meaning, show is a little bit wrong. What do you think?

Reply:
agree, done